man/confchpldoc.rst

```diff
@@ -1,5 +1,5 @@
 
-:Version: 2.1 pre-release
```

Comment:
eek, this is my bug in the release that it still says `pre-release` in the `chpldoc` man page.

Reply:
Ugh, do we need to respin?